ONSdigital/eq-survey-runner
# eq-153 Sass linting

### Changes
- Adds linting to Sass via [Stylelint](https://github.com/stylelint/stylelint).
- Linting config added to `.stylelintrc`.
- Rules follow [this standard](https://github.com/stylelint/stylelint-config-standard)
- Gulp task `lint:styles`
### Testing
- `npm install`
- run `gulp lint:styles`, confirm no errors or warnings
- open any `.scss` file in styles directory and add some code that will fail linting (eg. a bunch of newlines inside a rule)
- run `gulp lint:styles` again and confirm error/warning.
- remove offended style
- run `npm run compile` and confirm assets compile.
